render/persistentvolume.go

```diff
@@ -116,3 +134,67 @@ func (v pvTocontrollerRenderer) Render(rpt report.Report) Nodes {
 	}
 	return Nodes{Nodes: nodes}
 }
+
+// SPCToSPRenderer is a Renderer which produces a renderable kubernetes CRD SPC
+var SPCToSPRenderer = spcToSpRenderer{}
+
+//spcToSpRenderer is a Renderer to render SPC & SP nodes.
+type spcToSpRenderer struct{}
+
+//Render renders the SPC & SP nodes with adjacency.
+func (v spcToSpRenderer) Render(rpt report.Report) Nodes {
```

Comment:
On what basis the adjacency logic for SP & SPC is being written? A comment in the code would help!

Reply:
Also, comment at other `Renderer`'s  will help.